doc/source/vol-cli.rst

```diff
@@ -120,9 +120,7 @@ Options
     Change the default path used to store the cache.
 
 --offline
-    Do not search online for additional JSON files.
-    Run offline mode (defaults to false) and for
-    remote windows symbol tables, linux/mac banner repositories. 
+    Run offline mode (defaults to false).  Do not search online for additional JSON files, remote windows symbol tables, nor linux/mac banner repositories.
```

Comment:
Yep, I think this is right.  The goal is actually to shut down any use of the internet at all (for running in offline mode) and I *think* that's all the instances where it may be relevant, but I guess listing them explicitly is safer (because an user could include a URL in some way and the opener would try to go out to the URL automatically, whether `--offline` is set or not), so this is good.  5:)